### DISCUSSION TITLE
Application Testing

### DISCUSSION THREAD

**Discussion by MSarmadQadeer** [motis-project/ridesharing-flutter#159] [General]
I want to test the application. Can I have SUPABASE_BASE_URL and SUPABASE_BASE_KEY?

**Comment by fynsta** [comment]
The README is not entirely correct there, those credentials are only for the internal development team.

If you want to set the application up locally, you need to create your own Supabase instance. You can find the necessary files and instructions [here in the repository](https://github.com/motis-project/ridesharing-flutter/tree/main/supabase).
